gameci-1_intro.md

```diff
@@ -0,0 +1,146 @@
+Plus, it makes this nice visualization graph.
+
+## Pieces of the Workflow
+
+You may have noticed that there are a lot of different jobs in the workflow, but only some of them ran, while others did not run.
+Later, I'll explain all the different jobs and what they do/how they work, but the first thing to examine is why some jobs run while others do not.
+If you look at the names of the jobs that didn't run, you may be able to guess the reason: The jobs that deploy to production (the CD part of CI/CD) were not part of this workflow run.
+
+Depending on how you think of it, it could be argued that my workflow is actually multiple workflows in one.
+For every push to my develop branch, I run the test and build jobs, to confirm that that commit didn't break anything.
+When I want to actually deploy to production, I [create a release in GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release), which will trigger the workflow to run with all the production deployment jobs.
+I can also [manually run the workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow), with the option to input which jobs should run.
+
+The modularity of my workflow is enabled by setting up different triggers and blocking certain jobs with `if`s.
+You may refer to the [complete workflow on GitHub](https://github.com/finol-digital/Card-Game-Simulator/blob/develop/.github/workflows/main.yml), but going forward, I will excerpt the relevant code here.
+
+## The Code
+
+Let's get started with the start of my workflow:
+
+{% highlight yml %}
+{% raw %}
+# .github/workflows/main.yml
+name: Test, Build, and Deploy with GameCI
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - 'Assets/**'
+      - 'Packages/**'
+      - 'ProjectSettings/**'
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+    paths:
+      - 'Assets/**'
+      - 'Packages/**'
+      - 'ProjectSettings/**'
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      workflow_mode:
+        description: '[release] [Android, iOS, StandaloneLinux64, WebGL, StandaloneWindows, StandaloneWindows64, WSAPlayer, StandaloneOSX, Steam]'
+        required: false
+        default: ''
+jobs:
+{% endraw %}
+{% endhighlight %}
+
+## Trigger On
+
+As you can see, there are 4 triggers on: `push` to develop, `pull_request` to main, `release` published (with GitHub Release), and `workflow_dispatch`.
+
+The `push` and `pull_request` triggers should run checks to validate each commit, along with a final validation before merging changes to `main`.
+Runs on `push` and `pull_request` should only trigger if a file has changed in the actual Unity project, ie in `Assets/`, `Packages/`, or `ProjectSettings/`.
+Any changes outside of these folders would not cause our Unity project to change, so they can be ignored.
```

Comment:
Suggest saying either `my Unity project` or `the Unity project` to prevent moving the subject between being `mine` and `ours`.